packages/dx-react-grid/docs/guides/getting-started.md

```diff
@@ -32,6 +32,12 @@ This package does not contain visual components. In the examples below, visual c
 
   Make sure that [Material UI](https://material-ui-1dab0.firebaseapp.com/) dependencies are installed and properly configured. Check the Material UI's [Getting Started](https://material-ui-1dab0.firebaseapp.com/getting-started/installation) article for configuration details.
 
+### Supported Browsers
+
+React Grid supports the latest, stable releases of all major browsers: Chrome, Firefox, Safari, Opera, Microsoft Edge.
```

Comment:
major -> popular? or just '..the latest version of the following browsers:...'

Reply:
What is wrong with 'major browsers'?

Reply:
Just sounds strange to me.